src/electionguard/constants.py

```diff
@@ -0,0 +1,81 @@
+    115792089237316195423570985008687907853269984665640564039457584007913129639747,  # pow(2, 256) - 189
+    9019518416950528558373478086511232658951474842525520401496114928154304263969655687927867442562559311457926593510757267649063628681241064260953609180947464800958467390949485096429653122916928704841547265126247408167856620024815508684472819746384115369148322548696439327979752948311712506113890045287907335656308945630141969472484100558565879585476547782717283106837945923693806973017510492730838409381014701258202694245760602718602550739205297257940969992371799325870179746191672464736721424617639973324090288952006260483222894269928179970153634220390287255837625331668555933039199194619824375869291271098935000699785346405055160394688637074599519052655517388596327473273906029869030988064607361165803129718773877185415445291671089029845994683414682274353665003204293107284473196033588697845087556526514092678744031772226855409523354476737660407619436531080189837076164818131039104397776628128325247709678431023369197272126578394856752060591013812807437681624251867074769638052097737959472027002770963255207757153746376691827309573603635608169799503216990026029763868313819255248026666854405409059422844776556067163611304891154793770115766608153679099327786,
+    119359756198641231858139651428439585561105914902686985078252796680474637856752833978884422594516170665312423393830118608408063594508087813277769835084746883589963798527237870817233369094387978405585759195339509768803496494994109693743279157584139079471178850751266233150727771094796709619646350222242437970473900636242584673413224137139139346254912172628651028694427789523683070264102332413084663100402635889283790741342401259356660761075766365672754329863241692760862540151023800163269173550320623249398630247531924855997863109776955214403044727497968354022277828136634059011708099779241302941071701051050378539485717425482151777277387633806111112178267035315726401285294598397677116389893642725498831127977915200359151833767358091365292230363248410124916825814514852703770457024102738694375502049388804979035628232209959549199366986471874840784466132903083308458356458177839111623113116525230200791649979270165318729763550486200224695556789081331596212761936863634467236301450039399776963661755684863012396788149479256016157814129329192490798309248914535389650594573156725696657302152874510063002532052622638033113978672254680147128450265983503193865576932419282003012093526302631221491418211528781074474515924597472841036553107847,
+)
+
+# TEST ONLY
+# These constants serve as sets of primes for future developers
+# Currently, all the sets are all valid but may break certain tests
+# As tests adapt, these constants can be used to speed up tests
+EXTRA_SMALL_TEST_CONSTANTS = create_constants(157, 13, 12, 16)
+SMALL_TEST_CONSTANTS = create_constants(503, 251, 2, 5)
```

Comment:
I don't really see the point of teeny tiny tests like this. The "medium" test constants will run incredibly fast and will thus be more likely to uncover bugs, if they exist. (There are also some assumptions, in the tests, were we assume that collisions never ever happen, which are perfectly reasonable assumptions when we're dealing with 256-bit numbers that might as well be drawn randomly, but when I've experimentally tried these smaller constants, some tests fail. Those tests are still absolutely correct, they just shouldn't run for the tiny constants.)

Reply:
While I agree with that it is not good practice to use the tiny constants for the entire test suite. 

Having a couple sets of constants available to the public can still be helpful.The suite as it stands still takes 2-3 min to run. These constants could be used for a single test to help speed up with certain cases or if more tests are added. Perhaps even running a lighter test suite for PR's instead of Releases.